assets/src/scss/base/_mixins.scss

```diff
@@ -99,6 +99,15 @@
   left: 0;
 }
 
+// Move an element at the top-left corner of its nearest ancestor and set its width and height to 100%.
+@mixin absolute-position {
+  position: absolute;
+  top: 0;
+  left: 0;
```

Comment:
I'm wondering if for RTL sites this should be `right` instead?

Reply:
Although it probably doesn't matter since it's full width/height

Reply:
It will always be full width and height :)